[ci-visibility] Improve cypress docs (#21928)
DataDog/documentation: content/en/tests/setup/javascript.md

### Cypress before version 10

> These are the instructions if you're using a version older than `cypress@10`. See more about migrating to a newer version in [cypress docs][9]

Reviewer: ```suggestion
These are the instructions if you're using a version older than `cypress@10`. See the [Cypress documentation][9] for more information about migrating to a newer version.
```

Reviewer: I'm gonna merge this and then merge in master to fix the link check
